CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Braintree iOS SDK Release Notes
 
 ## unreleased
+* Send `start_time`, `end_time`, and `endpoint` to FPTI for tracking API request latency
```

Comment:
We can move this under `BraintreeCore`

Reply:
yes, we can [4607716](https://github.com/braintree/braintree_ios/pull/1321/commits/460771682793594fef03f858902cdf4286d0b6c1)